app/Models/File.php

```diff
@@ -151,17 +151,23 @@ public function getInfo()
     /**
      * Sync the song with all available media info against the database.
      *
-     * @param array $tags  The (selective) tags to sync (if the song exists)
-     * @param bool  $force Whether to force syncing, even if the file is unchaged
+     * @param array $tags       The (selective) tags to sync (if the song exists)
+     * @param bool  $forceFiles Whether to force syncing, even if the file is unchanged
+     * @param bool  $forcePath  Whether to force syncing of a specific path, even
+     *                          if the file is unchanged
      *
      * @return bool|Song A Song object on success,
      *                   true if file exists but is unmodified,
      *                   or false on an error.
      */
-    public function sync($tags, $force = false)
+    public function sync($tags, $forceFiles = false, $forcePath = null)
```

Comment:
Top of my head, `$forceFiles` can be kept as `$force`, and `$forcePath` can be simply `$path` – that should be self-explanatory enough.